### PR TITLE
Intent slot - adding Jenkins test

### PR DESCRIPTION
**Pull request by vladgets**
Signed-off-by: Vlad Getselevich <vgetselevich@nvidia.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -424,6 +425,26 @@ pipeline {
       }
     }
 
+    stage('L2: Intent and Slot Classification') {
+      when {
```
Comment (VahidooX) [inline Jenkinsfile:429]: What does it do? the when and anyOf?

File: nemo/collections/nlp/models/intent_slot_classification_model.py [hunk 2]
```diff
@@ -224,7 +224,6 @@ def _setup_dataloader_from_config(self, cfg: DictConfig):
                  of Intents and Slots files.'
             )
 
-        # TODO: check that file exists
         dataset = IntentSlotClassificationDataset(
```
Comment (VahidooX) [inline nemo/collections/nlp/models/intent_slot_classification_model.py:227]: Do you check it dataset file exists?

Comment (vladgets) [reply nemo/collections/nlp/models/intent_slot_classification_model.py:227]: Yes, I do in rows before that. I just removed old TODO comment here.

File: Jenkinsfile (hunk 1 above)
Comment (vladgets) [reply Jenkinsfile:429]: It is not clear. I had to resolve merge conflict in this file manually. Jenkins code seems to be fine, but in the test it run Green only until the middle and then stopped for some unclear reason.

**Pull request merged**